modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -47,17 +46,17 @@ for every size and can be positioned at the `start` or `end` with the `iconPosit
 
 <ExampleCodeBlock code={Primary} />
 
+---
+
 Primary Buttons also have an `inverse` variant. While it looks similar to the default Secondary
 Button, the default outline as well as the hover and focus states are different. Use this variant
 when you need to place a Primary Button on a dark or colorful background such as `blueberry400`.
 
 <ExampleCodeBlock code={PrimaryInverse} />
 
-#### Props
-
-Undocumented props are spread to the underlying `<button>` element.
+### Component API
```

Comment:
_This feedback applies to the PrimaryButton, SecondaryButton, TertiaryButton, and DeleteButton sections in the Button MDX._

The `Component API` should be an h4 since it's a subsection of `### PrimaryButton`. I think this means we would also need to use the `headingStart` prop of the `SymbolDoc` for PrimaryButton to ensure the headings rendered by `SymbolDoc` are at the right heading level.

Reply:
:+1: